content/en/docs/examples/virtual-machines/bookinfo/index.md

```diff
@@ -12,6 +12,151 @@ aliases:
 - /docs/examples/vm-bookinfo
 ---
 
+## Send requests from VM workloads to Kubernetes services
+
+After setup, the machine can access services running in the Kubernetes cluster
+or on other VMs.
+
+The following example shows accessing a service running in the Kubernetes cluster from a VM using
+`/etc/hosts/`, in this case using a service from the [Bookinfo example](/docs/examples/bookinfo/).
+
+1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
+
+    {{< text bash >}}
+    $ kubectl get svc productpage -o jsonpath='{.spec.clusterIP}'
+    10.55.246.247
+    {{< /text >}}
+
+1.  Then on the added VM, add the service name and address to its `etc/hosts`
+    file. You can then connect to the cluster service from the VM, as in the
+    example below:
+
+    {{< text bash >}}
+$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
+$ curl -v productpage.default.svc.cluster.local:9080
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 1836
+< server: envoy
+... html content ...
+    {{< /text >}}
+
+The `server: envoy` header indicates that the sidecar intercepted the traffic.
+
+## Running services on the added VM
+
+1. Setup an HTTP server on the VM instance to serve HTTP traffic on port 8080:
+
+    {{< text bash >}}
+    $ gcloud compute ssh ${GCE_NAME}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Determine the VM instance's IP address. For example, find the IP address
+    of the GCE instance with the following commands:
```

Comment:
```suggestion
1. Determine the VM instance's IP address. For example, to find the IP address
    of the GCE instance, use the following commands:
```
(Fine to leave it as you have it, but I think it's a little clearer with the "to".)